WordPress/gutenberg
# Site Editor: Hide export button if non-block-based theme

## What?

This PR hides the theme's export button when it is not a block theme.

![image](https://github.com/WordPress/gutenberg/assets/54422211/4f2f61e9-cdcd-49d9-9bb8-436289476c44)

## Why?

My understanding is that this feature, as the text suggests, exports updated templates and global styles. However, since neither templates nor global styles are used in the classic theme, I don't think they should be displayed.

## How?

Hides the button if it's not a block theme.

## Testing Instructions

- Access `localhost:8889/wp-admin`
- Activate Emptyhybrid
- Access Appearance > Template Parts > Header > Editor canavs > Options
- The export button should not be visible
